client/components/Tests/TestGen.js

```diff
@@ -401,7 +406,7 @@ export const Editor = (props) => {
               Evaluate Test
             </button>
             <button
-              className='filled-button self-center rounded-lg bg-lime-400 px-4 py-2 text-sm text-slate-900 transition-shadow 2xl:text-base'
+              className='filled-button self-center rounded-lg bg-lime-400 px-4 py-2 text-sm  text-slate-900 transition-shadow 2xl:text-base'

```

Comment:
Test 1 and 3 seem to have an extra space in the Submit Test button in classes.
just a format difference